rclpy/rclpy/destroyable.py

```diff
@@ -17,7 +17,6 @@
 
 
 class DestroyableType(Protocol):
-
```

Comment:
https://ci.ros2.org/job/ci_linux/20767/testReport/ needs this line, and this is unrelated change.

Reply:
done